src/Control/Carrier.hs

```diff
@@ -9,7 +9,7 @@ module Control.Carrier
 , send
 ) where
 
-import {-# SOURCE #-} Control.Carrier.Class
+import Control.Carrier.Class
```

Comment:
I was running into some weird compilation errors, and it occurred to me that I’d put the `.hs-boot` files on the wrong edges. Putting them on the `Control.Effect.*` types allows me to limit the boot files to just the datatypes & instances needed to define the `base` & `transformers` instances.